.github/workflows/playwright.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       with:
         node-version: 18
     - name: Install dependencies
-      run: npm ci
```

Comment:
Don't we want to keep this? We made these changes so local development is less confusion.

Reply:
Yeah you're right - this is not correct. I just fixed them and pushed, but now it says there are no differences between the branches. I think I will close this for now, and we can sort out the differences between main and dev/unified-sdk when we release the next version.